hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/SCMPipelineManager.java

```diff
@@ -115,6 +114,12 @@ public SCMPipelineManager(Configuration conf, NodeManager nodeManager,
         "SCMPipelineManagerInfo", this);
     initializePipelineState();
     this.grpcTlsConfig = grpcTlsConfig;
+    this.pipelineNumberLimit = conf.getInt(
+        ScmConfigKeys.OZONE_SCM_PIPELINE_NUMBER_LIMIT,
+        ScmConfigKeys.OZONE_SCM_PIPELINE_NUMBER_LIMIT_DEFAULT);
+    this.heavyNodeCriteria = conf.getInt(
```

Comment:
NIT: suggest rename heavyNodeCriteria to maxPipelinePerDatanode. heavyNodeCriteria can be a function name using maxPipelinePerDatanode.

Reply:
Sure